src/models/newApplicant.ts

```diff
@@ -0,0 +1,21 @@
+export interface NewApplicant {
+  discordUsername?: string;
+  name: string;
+  email: string;
+  age: number;
+  skills: {
+    id?: string;
+    skill: string;
+  }[];
+  commitment: number;
+  extraInfo?: string;
+  portfolioLink?: string;
+  experience: string;
+  reason: string;
+  accepted: boolean;
+  zip: number;
+  role: string;
+  id: string;
+  project: string;
+  level: number;
```

Comment:
level is not a number, you can import https://github.com/dev-launchers/Dev-Recruiters/blob/0a0ec2f6fa546bbcce8348563765f9db8f7f3d93/src/models/level.ts and used it instead of number